XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -1133,7 +1168,23 @@ private void toggleConsoleButton_Click(object sender, EventArgs e)
 
                     vncScreen.UseVNC = !vncScreen.UseVNC;
                     vncScreen.UserWantsToSwitchProtocol = true;

```

Comment:
Yes, I think it is because user made choice of using which kind of connection already. It will affect the behavior of "Automatically switch to the Remote Desktop console when it becomes available" which is parameter "Properties.Settings.Default.AutoSwitchToRDP".
If this UserWantsToSwitchProtocol has been set, we will not try to automatically switch to Remote Desktop console.